jupyter_collaboration/ychat.py

```diff
@@ -1,15 +1,16 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
+# TODO: remove this module in favor of the one in jupyter_ydoc when released.
+
 import json
+import jupyter_ydoc
 from functools import partial
 from typing import Any, Callable, List
-
-from jupyter_ydoc.ybasedoc import YBaseDoc
 from pycrdt import Array, Map
 
 
-class YChat(YBaseDoc):
+class YChat(jupyter_ydoc.YBaseDoc):
```

Comment:
I don't understand how that can work, since `jupyter_ydoc` doesn't export `YBaseDoc`:
```py
import jupyter_ydoc

jupyter_ydoc.YBaseDoc
# Traceback (most recent call last):
#   File "<stdin>", line 1, in <module>
# AttributeError: module 'jupyter_ydoc' has no attribute 'YBaseDoc'. Did you mean: 'ybasedoc'?
```

Reply:
Me neither, but i fixed it anyway.

Reply:
Maybe the `YBaseDoc` should be prefixed with an underscore, to specified that it should not be imported.

Reply:
But it's fine to import it, right?

Reply:
I think so, I misunderstood your comment and thought you meant not to import it.